package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-cucumberjs",
   "description": "Generates documentation from Cucumber features",
-  "version": "0.9.2",
+  "version": "0.9.3",
```

Comment:
@SirLenz0rlot Can you un-commit the versioning? We will bump the version with `npm version` to have the commit with versioning.